neo/api/JSONRPC/JsonRpcApi.py

```diff
@@ -332,3 +339,21 @@ def get_peers(self):
                                               "port": int(port)})
 
         return result
+
+    def list_address(self):
+        """Get information about all the addresses present on the open wallet
+
+        The current implementation still does not fully comply with the spec,
+        since it seems there isn't a straight forward way of getting "haskey"
+        information.
+        """
+        result = []
+        for addrStr in self.wallet.Addresses:
+            addr = self.wallet.GetAddress(addrStr)
+            result.append({
+                "address": addrStr,
+                # "haskey": True,
```

Comment:
Could this just be:

```
"haskey": not addr.IsWatchOnly
```

?

Reply:
@localhuman @metachris what do you think? 
Just need to make sure that information will be correct.

Reply:
I think its a good idea